src/Simulations/simulation.jl

```diff
@@ -38,7 +35,13 @@ function Simulation(model::AbstractModel, method; Y_init = nothing, dt, tspan)
 end
 
 """
-    set!
+    set!(
+        simulation::AbstractSimulation,
+        submodel_name = nothing;
+        kwargs...,
+    )
+
+Set up the `simulation`.
```

Comment:
This sets the simulation state to a new state, either through an array or a function. This is signified by the "!" after the "set", aka it updates the existing simulation instance that is passed as first argument.